src/_includes/cloud/note-pro-elasticsearch-version-change.md

```diff
@@ -0,0 +1,2 @@
+ {:.bs-callout-info}
+For Pro projects, you must create a [Support ticket](http://support.magento.com) to change the Elasticsearch version installed on Staging and Production environments. It can take up to 48 hours for the Magento infrastructure team to update your project to the requested service version.
```

Comment:
This note is so similar to the other note. The only difference is the other note is more generic and references the YAML files. The services YAML just lists the services and their version, so I'm not clear on why Elasticsearch needs a separate note. The service change needed would be to update the <service-name> version to <version-number> in the remote environments. Pretty generic.

Reply:
so there's an associated MCLOUD ticket that specifically requested adding the note about the support ticket to Elasticsearch.  Based on that, I ended up modifying the generic services note after adding this note specific to Elasticsearch.

I think you are correct that I could use the same note here on the Elasticsearch topic, rather than adding a separate one.